stack/stack.go

```diff
@@ -1,23 +1,43 @@
+// Package stack proporciona una implementación de una pila genérica sobre un arreglo dinámico.
+// Expone la estructura y funciones básicas para manipular una pila.
 package stack
 
 import "errors"
 
-// Stack implementa una pila genérica sobre un arreglo dinámico.
+// Proporciona una pila cuyos elementos son de un tipo genérico.
+// La implementación se basa en un arreglo dinámico.
```

Comment:
La documentación de funciones y métodos tienen que empezar con el nombre del mismo:

```suggestion
// Stack proporciona una pila cuyos elementos son de un tipo genérico.
// La implementación se basa en un arreglo dinámico.
```

Reply:
ok, entonces restauro todas las que saqué :)
Gracias por el dato!

(no puedo ver un preview hasta que se mergee en master)

Reply:
No estoy encontrando el lugar específico donde dice que es la "sintaxis" de los comments, pero creo recordar que era sacado de acá: https://tip.golang.org/doc/comment